examples/gogo/command/src/main/java/org/osgi/enroute/examples/gogo/command/ComponentMulti.java

```diff
@@ -0,0 +1,28 @@
+
+package org.osgi.enroute.examples.gogo.command;
+
+import org.osgi.service.component.annotations.Component;
+
+@Component(
+        /* Activate the component immediately, also if no one depends on it. */
+        immediate = true,
```

Comment:
There is no reason to make this component immediate, and it is a bad practice to make a component immediate unless it really needs to be.

Reply:
Hm, I could be wrong, but I used Karaf and Apache Felix the last years and if a component is not immediate and no one consumes the service the component provides the component gets not activated.

Reply:
The component won't be activated eagerly, but it will be registered as a service. The component will then be activated when someone uses the service.

Reply:
Thanks for clarification.